app/views/includes/profile-header.nunjucks

```diff
@@ -1,16 +1,14 @@
+<p class="gp-address">{{ gp.address.addressLines | safe | replace(",", ", ") }}, {{ gp.address.postcode }}</p>
+<p><a href="{{mapUrl}}">See map and get directions</a></p>
 <div class="local-header__intro local-header__intro--fullwidth">
   <div class="grid-row">
     <div class="column--one-half">
-      <p>{{ gp.address.addressLines | safe | replace(",", "<br />") }}<br />{{ gp.address.postcode }}</p>
-      <a href="{{mapUrl}}">See map and get directions</a>
+      <p>{% if gp.contact.telephone %}Reception: {{ gp.contact.telephone }}{% endif %}</p>
```

Comment:
`{% if gp.contact.telephone %}<p>Reception: {{ gp.contact.telephone }}</p>{% endif %}`

We should be putting the condition outside of the `<p>` to avoid empty elements.